:farmer: Rolling and Jazzy performance jobs failing

Jazzy and Rolling performance jobs are failing to build:
* [Rolling Performance](https://build.ros2.org/view/Rci/job/Rci__nightly-performance_ubuntu_noble_amd64/19/consoleFull)
* [Jazzy Performance](https://build.ros2.org/view/Jci/job/Jci__nightly-performance_ubuntu_noble_amd64/8/consoleFull)

Console output for both show compilation errors in `experiment_configuration/qos_abstraction.cpp`:

```
00:09:39.400 In file included from /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.cpp:15:
00:09:39.400 /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.hpp:61:3: error: ‘uint32_t’ does not name a type
00:09:39.400    61 |   uint32_t history_depth;
00:09:39.400       |   ^~~~~~~~
00:09:39.400 /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.hpp:19:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
00:09:39.400    18 | #include <ostream>
00:09:39.400   +++ |+#include <cstdint>
00:09:39.400    19 | 
00:09:39.400 /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.hpp: In constructor ‘performance_test::QOSAbstraction::QOSAbstraction()’:
00:09:39.400 /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.hpp:32:5: error: class ‘performance_test::QOSAbstraction’ does not have any field named ‘history_depth’
00:09:39.400    32 |     history_depth(),
00:09:39.400       |     ^~~~~~~~~~~~~
00:09:39.400 /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.cpp: In function ‘std::ostream& performance_test::operator<<(std::ostream&, QOSAbstraction)’:
00:09:39.400 /tmp/ws2/src/performance_test/performance_test/src/experiment_configuration/qos_abstraction.cpp:59:34: error: ‘const struct performance_test::QOSAbstraction’ has no member named ‘history_depth’
00:09:39.400    59 |          " History depth: " << e.history_depth <<
00:09:39.400       |                                  ^~~~~~~~~~~~~
00:09:39.400 gmake[2]: *** [CMakeFiles/perf_test.dir/build.make:150: CMakeFiles/perf_test.dir/src/experiment_configuration/qos_abstraction.cpp.o] Error 1
00:09:39.400 gmake[1]: *** [CMakeFiles/Makefile2:663: CMakeFiles/perf_test.dir/all] Error 2
00:09:39.400 gmake: *** [Makefile:146: all] Error 2
00:09:39.400 ---
00:09:39.400 Failed   <<< performance_test [6min 14s, exited with code 2]
```

--- a/performance_test/src/data_running/data_runner.hpp
+++ b/performance_test/src/data_running/data_runner.hpp
@@ -210,9 +210,9 @@ private:
     m_memory_tools_on = true;
     #endif
   }
+  SpinLock m_lock;
   TCommunicator m_com;
   std::atomic<bool> m_run;
-  SpinLock m_lock;
 
   uint64_t m_sum_received_samples;
   uint64_t m_sum_lost_samples;

--- a/performance_test/src/experiment_configuration/qos_abstraction.hpp
+++ b/performance_test/src/experiment_configuration/qos_abstraction.hpp
@@ -15,6 +15,7 @@
 #ifndef EXPERIMENT_CONFIGURATION__QOS_ABSTRACTION_HPP_
 #define EXPERIMENT_CONFIGURATION__QOS_ABSTRACTION_HPP_
 
+#include <cstdint>
 #include <ostream>
 
 namespace performance_test

